install.sh

```diff
@@ -1,25 +1,51 @@
+#!/bin/bash
+
+###############################################################################
+# Instalation Script for Gitshots
+# Repository: https://github.com/ranman/gitshots-client
+# gitshots.com - Take a picture and collect some stats every time you commit!
+###############################################################################
+
+USERHOME=$(dscl . -read /Users/`whoami` NFSHomeDirectory | awk -F": " '{print $2}')
```

Comment:
why not just rely on the environment variable `$HOME` since we're already relying on `whoami`